pkg/controller/controller.go

```diff
@@ -19,15 +19,14 @@ package controller
 
 import (
 	"fmt"
-	"sync"
-	"time"
-
 	machineinternal "github.com/gardener/machine-controller-manager/pkg/apis/machine"
 	machinev1alpha1 "github.com/gardener/machine-controller-manager/pkg/apis/machine/v1alpha1"
 	machinescheme "github.com/gardener/machine-controller-manager/pkg/client/clientset/versioned/scheme"
 	machineapi "github.com/gardener/machine-controller-manager/pkg/client/clientset/versioned/typed/machine/v1alpha1"
 	machineinformers "github.com/gardener/machine-controller-manager/pkg/client/informers/externalversions/machine/v1alpha1"
 	machinelisters "github.com/gardener/machine-controller-manager/pkg/client/listers/machine/v1alpha1"
+	"github.com/gardener/machine-controller-manager/pkg/util/worker"
+	"sync"
```

Comment:
pls keep the "sync" package below "fmt" under import.
We usually try to follow the following grouping of imports
- any in-built libraires
- any same repo packages
- any package from third party

Reply:
Cool. this formatting was done by Intellij. What I did to fix this according was go to _Settings>Editor > Code Style > Go > Imports_ and checked the "Group > Current Project Packages" and "Group stdlib imports" option. That appears to have fixed this.